<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>655</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Matriz A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantidade de Linhas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantidade de Colunas</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="QLinhas">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="QColunas">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="SaidaMA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCriarMA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Criar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantidade de Linhas</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="QLinhasB">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantidade de Colunas</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="QColunasB">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCriarMatrizB">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Criar</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="SaidaMB">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>181</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="soma">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LimparB">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>230</y>
      <width>90</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="SaidaMC">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>201</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Matriz B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="subtracao">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LimparA">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LimparC">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>240</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multiplicacao">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Matriz Resultante</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multiplicarK">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>A*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="K">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="igualdade">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>290</y>
      <width>381</width>
      <height>211</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <row>
     <property name="text">
      <string>Triangular Superior</string>
     </property>
     <property name="background">
      <color>
       <red>170</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Triangular Inferior</string>
     </property>
     <property name="background">
      <color>
       <red>170</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Matriz Ortogonal</string>
     </property>
     <property name="background">
      <color>
       <red>170</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Matriz Simetrica</string>
     </property>
     <property name="background">
      <color>
       <red>170</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Matriz Identidade</string>
     </property>
     <property name="background">
      <color>
       <red>170</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Matriz de Permutação</string>
     </property>
     <property name="background">
      <color>
       <red>170</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Matriz A</string>
     </property>
     <property name="background">
      <color>
       <red>170</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Matriz B</string>
     </property>
     <property name="background">
      <color>
       <red>170</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="transpostaA">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="transpostaB">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>410</y>
      <width>171</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="multiplicarKB">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>B*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="KB">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>280</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="potenciaA">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>A^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="potenciaB">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>B^</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="potA">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="potB">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>340</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>!=</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Transposta de A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>380</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Transposta de B</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>868</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMatrizwe">
    <property name="title">
     <string>Matrizes</string>
    </property>
   </widget>
   <addaction name="menuMatrizwe"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
